checks.d/mysql.py

```diff
@@ -263,13 +585,24 @@ def _get_version(self, db, host):
         cursor.close()
```

Comment:
Let's use Context Managers to properly close all our cursors:
http://stackoverflow.com/questions/8067690/context-manager-for-pythons-mysqldb/22840557#22840557